### ISSUE TITLE
Support for Firebase Hosting

### ISSUE THREAD

**Issue by JosefJezek** [pixelcodeUK/polymer-static-analyze#2]
Could you add support for Firebase Hosting and `nopush` directive? 

https://firebase.googleblog.com/2016/09/http2-comes-to-firebase-hosting.html

**Comment by pixelcodeUK** [comment]
Yeah, I'll add an option to build a `firebase.json` compatible entry

**Comment by pixelcodeUK** [comment]
@JosefJezek I've made a new branch here:

https://github.com/pixelcodeUK/polymer-static-analyze/tree/feature/firebase

It has V1 of the `Firebase.json` file generation. The only problem is it requires the main routes of the app which the `Polymer.json` doesn't have, so I have split the routes one for each fragment.

It automatically adds `rel=preload` for files that are not heavily referenced, those that are (such as Polymer) as pushed as normal.
